Content.Server/Atmos/AtmosHelpers.cs

```diff
@@ -11,18 +12,18 @@ namespace Content.Server.Atmos
 {
     public static class AtmosHelpers
     {
-        public static TileAtmosphere? GetTileAtmosphere(this EntityCoordinates coordinates, IEntityManager? entityManager = null)
+        public static TileAtmosphere GetTileAtmosphere(this EntityCoordinates coordinates, IEntityManager? entityManager = null)
         {
             entityManager ??= IoCManager.Resolve<IEntityManager>();
 
             var gridAtmos = EntitySystem.Get<AtmosphereSystem>().GetGridAtmosphere(coordinates.GetGridId(entityManager));
 
-            return gridAtmos?.GetTile(coordinates);
+            return gridAtmos.GetTile(coordinates);
```

Comment:
What happens if the grid doesn't have atmos?

Reply:
It returns a SpaceAtmosphereComponent.

https://discord.com/channels/310555209753690112/560845886263918612/783748700588867614